packages/eslint-config-kibana/.eslintrc.js

```diff
@@ -10,8 +10,13 @@ module.exports = {
     'import',
     'no-unsanitized',
     'prefer-object-spread',
+    'jsx-a11y',
   ],
 
+  // "extends": [
```

Comment:
__nit:__ please remove commented code before merging

__question:__ what is the reason why we don't want to use `recommended` preset btw?

Reply:
@azasypkin the recommended preset has some deprecated rules which Tim and I want to rewrite.